src/app/core/services/auth.service.spec.ts

```diff
@@ -0,0 +1,107 @@
+import { TestBed } from '@angular/core/testing';
+
+import { AuthService } from './auth.service';
+
+describe('service', () => {
```

Comment:
It's better to write class name into describe

`describe('AuthService', () => {})`